src/main/java/org/synyx/urlaubsverwaltung/calendar/ICalService.java

```diff
@@ -101,7 +101,7 @@ private Optional<VEvent> toVEvent(Absence absence, ICalType method, boolean isOw
             }
             event.add(new XProperty("X-MICROSOFT-CDO-ALLDAYEVENT", "TRUE"));
         } else {
-            event = new VEvent(startDateTime.toOffsetDateTime(), endDateTime.toOffsetDateTime(), absence.getEventSubject());
+            event = new VEvent(startDateTime.toInstant(), endDateTime.toInstant(), absence.getEventSubject());
```

Comment:
Nitpick: The structure and readability would benefit, if `Absence` had `getStartInstant` and `getEndInstant` to use here and `getStartLocalDate` and `getEndLocalDate` for L97-L100

Reply:
I think we need to rethink the "new" calender absence class, and then we will take this into account.